frontend/public/components/secret.jsx

```diff
@@ -12,9 +12,9 @@ kind: Secret
 metadata:
   name: example
 type: Opaque
-data:
-  username: YWRtaW4=
-  password: MWYyZDFlMmU2N2Rm`);
+stringData:
+  username: admin
```

Comment:
Should we provide a dummy password that is *clearly* not a generated password? 
This looks legitimate enough that I could see someone accepting it.

Reply:
Yeah, I decoded the existing values from the template, but that's a better idea

Reply:
😂